docs/content/preview/explore/query-1-performance/ycql-stat-statements.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Get query statistics using ycql_stat_statements
+linkTitle: Get query statistics
+description: Track planning and execution statistics for all YCQL statements executed by a server.
+headerTitle: Get query statistics using ycql_stat_statements
+menu:
+  preview:
+    identifier: ycql-stat-statements
+    parent: query-tuning
+    weight: 200
+type: docs
+---
+
+{{<tabs>}}
+{{<tabitem href="../pg-stat-statements/" text="YSQL" icon="postgres" >}}
+{{<tabitem href="../ycql-stat-statements/" text="YCQL" icon="cassandra" active="true" >}}
+{{</tabs>}}
+
+Databases can be resource-intensive, consuming a lot of memory CPU, IO, and network resources. Optimizing your CQL can be very helpful in minimizing resource utilization. The `ycql_stat_statements` module helps you track planning and execution statistics for all the YCQL statements executed by a server.
+
+This view provides YCQL statement metrics (similar to pg_stat_statements) that are also present on `<yb-tserver-ip>:12000/statements`, accessible via YSQL. The view can be joined with YCQL wait events in the [yb_active_session_history](../../observability/active-session-history/#yb-active-session-history) view using the query ID.
```

Comment:
This view provides YCQL statement metrics (similar to pg_stat_statements) that are also present on `<yb-tserver-ip>:12000/statements`, accessible via YSQL

Please remove ", accessible via YSQL. It is not making sense. This is a YCQL endpoint only.

Reply:
Oh I added it based on this response : https://docs.google.com/document/d/1VRiPYWTclpxBnyxxix9yD87WIZQrsALviDY5qZXha9Y/edit?disco=AAABMAeLnqA
@hbhanawat